eqasc/code/environment.yml

```diff
@@ -0,0 +1,114 @@
+name: grc
+channels:
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - ca-certificates=2020.7.22=0
+  - certifi=2020.6.20=py37_0
+  - libedit=3.1.20191231=h14c3975_1
+  - libffi=3.2.1=hd88cf55_4
+  - libgcc-ng=9.1.0=hdf63c60_0
+  - libstdcxx-ng=9.1.0=hdf63c60_0
+  - ncurses=6.2=he6710b0_1
+  - openssl=1.1.1g=h7b6447c_0
+  - pip=20.2.2=py37_0
+  - python=3.7.5=h0371630_0
+  - readline=7.0=h7b6447c_5
+  - setuptools=49.6.0=py37_0
+  - sqlite=3.33.0=h62c20be_0
+  - tk=8.6.10=hbc83047_0
+  - wheel=0.35.1=py_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - alabaster==0.7.12
+    - allennlp==0.9.0
+    - attrs==20.2.0
+    - babel==2.8.0
+    - blis==0.2.4
+    - boto3==1.14.62
+    - botocore==1.17.62
+    - chardet==3.0.4
+    - click==7.1.2
+    - conllu==1.3.1
+    - cycler==0.10.0
+    - cymem==2.0.3
+    - docutils==0.16
+    - editdistance==0.5.3
+    - flaky==3.7.0
+    - flask==1.1.2
+    - flask-cors==3.0.9
+    - ftfy==5.8
+    - future==0.18.2
+    - gevent==20.6.2
+    - greenlet==0.4.16
+    - h5py==2.10.0
+    - idna==2.10
+    - imagesize==1.2.0
+    - importlib-metadata==1.7.0
+    - iniconfig==1.0.1
+    - itsdangerous==1.1.0
+    - jinja2==2.11.2
+    - jmespath==0.10.0
+    - joblib==0.16.0
+    - jsonnet==0.16.0
+    - jsonpickle==1.4.1
+    - kiwisolver==1.2.0
+    - markupsafe==1.1.1
+    - matplotlib==3.3.2
+    - more-itertools==8.5.0
+    - murmurhash==1.0.2
+    - nltk==3.5
+    - numpy==1.19.2
+    - numpydoc==1.1.0
+    - overrides==3.1.0
+    - packaging==20.4
+    - parsimonious==0.8.1
+    - pillow==7.2.0
+    - plac==0.9.6
+    - pluggy==0.13.1
+    - preshed==2.0.1
+    - protobuf==3.13.0
+    - py==1.9.0
+    - pygments==2.7.0
+    - pyparsing==2.4.7
+    - pytest==6.0.2
+    - python-dateutil==2.8.1
+    - pytorch-pretrained-bert==0.6.2
+    - pytorch-transformers==1.1.0
+    - pytz==2020.1
+    - regex==2020.7.14
+    - requests==2.24.0
+    - responses==0.12.0
+    - s3transfer==0.3.3
+    - scikit-learn==0.23.2
+    - scipy==1.5.2
+    - sentencepiece==0.1.91
+    - six==1.15.0
+    - snowballstemmer==2.0.0
+    - spacy==2.1.9
+    - sphinx==3.2.1
+    - sphinxcontrib-applehelp==1.0.2
+    - sphinxcontrib-devhelp==1.0.2
+    - sphinxcontrib-htmlhelp==1.0.3
+    - sphinxcontrib-jsmath==1.0.1
+    - sphinxcontrib-qthelp==1.0.3
+    - sphinxcontrib-serializinghtml==1.1.4
+    - sqlparse==0.3.1
+    - srsly==1.0.2
+    - tensorboardx==2.1
+    - thinc==7.0.8
+    - threadpoolctl==2.1.0
+    - toml==0.10.1
+    - torch==1.6.0
+    - tqdm==4.49.0
+    - unidecode==1.1.1
+    - urllib3==1.25.10
+    - wasabi==0.8.0
+    - wcwidth==0.2.5
+    - werkzeug==1.0.1
+    - word2number==1.1
+    - zipp==3.1.0
+    - zope-event==4.4
+    - zope-interface==5.1.0
+prefix: /home/jharsh/miniconda3/envs/grc
```

Comment:
I don't use Conda, but we shouldn't this line!

Reply:
Yeah agreed, it shouldn't be there. Will remove it.

Reply:
Since I got the code to use just numpy==1.19.2 and scikit-learn==0.23.2 with Python 3.6.8, your environment.yml file should shrink substantially.